### PR TITLE
issue/1236 - improve dataset page

### PR DESCRIPTION
**Pull request by sukhrajghuman**
1. Removed unneccessary underscores from file names
2. Made file and API names wrap in screen.
3. Made Modal fit better on smaller screen
4. Improved design of modal
5. Removed underlines from filenames.
6. Added Animation for modal opening
7. Added blur when opening modal

Modal fits better on mobile now. Though the feedback form button I
believe is
stuffing up the body grid.

### Checklist

-   [x] There are unit tests to verify my changes are correct | Unit tests aren't applicable (delete one)
-   [x] I've updated CHANGES.md with what I changed.
-   [x] I've linked this PR to an issue in ZenHub (or there is no issue) and dragged it to the _Pull Request_ column
-   [x] There are sufficient comments for my code to be understandable - and I realise reviewers will pull me up on it if not!

### CODE REVIEW

File: magda-web-client/src/Components/Dataset/DatasetSuggestForm.scss [hunk 1]
```diff
@@ -2,13 +2,17 @@
...
 .ask-dataset-form {
     padding: 10px;
-    border: solid;
-    border-color: $AU-color-foreground-focus;
-    background-color: $AU-colordark-foreground-action;
     overflow: -webkit-paged-y;
     position: relative;
     float: right;
     width: 800px;
+    background-color: rgb(248, 247, 247);
```
Comment (chloeleichen) [inline magda-web-client/src/Components/Dataset/DatasetSuggestForm.scss]: Can we use $AU-colordark-foreground-action instead? it is rgb(241,247,238) which is very close to this color here and can help keep the theme consistent

Comment (sukhrajghuman) [reply magda-web-client/src/Components/Dataset/DatasetSuggestForm.scss]: hi @chloeleichen  that color looks really bad. Please see below

image
****

Comment (chloeleichen) [reply magda-web-client/src/Components/Dataset/DatasetSuggestForm.scss]: yeah, i agree. Happy to use the new color 💃

Comment (sukhrajghuman) [reply magda-web-client/src/Components/Dataset/DatasetSuggestForm.scss]: 😸

**Pull request merged**